plugins/Sf2Player/CMakeLists.txt

```diff
@@ -3,8 +3,8 @@ if(LMMS_HAVE_FLUIDSYNTH)
 	build_plugin(sf2player
 		Sf2Player.cpp Sf2Player.h PatchesDialog.cpp PatchesDialog.h PatchesDialog.ui
 		MOCFILES Sf2Player.h PatchesDialog.h
-		UICFILES PatchesDialog.ui
 		EMBEDDED_RESOURCES *.png
 	)
-	target_link_libraries(sf2player fluidsynth SampleRate::samplerate)
+	set_target_properties(sf2player PROPERTIES AUTOUIC_SEARCH_PATHS ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
I still think this line, and the equivalent in `GigPlayer` can be removed. I've removed this one locally, and it builds without problems. Are you sure the error you got was due to this line being missing, and not some other problem?

Reply:
removed it.